.github/workflows/k8s-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           # are tested (https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#available-versions)
           - databases: pgsql
             brokers: redis
-            k8s: 'v1.26.11'
+            k8s: 'v1.30.3'
```

Comment:
I notice minikube is on [`v1.33.1`](https://github.com/kubernetes/minikube/releases/tag/v1.33.1) now. We should probably try to bump that as well (line 40).

Reply:
I agree. Done